pmg/sync_soundcloud.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from pmg import db
+from pmg.models import *
+from pmg.models.soundcloud import SoundcloudTrack
+from sqlalchemy import desc
+import soundcloud
+
+logger = logging.getLogger(__name__)
+
+
+def get_unstarted_query():
+    return db.session.query(File, Event.title.label('event_title')) \
```

Comment:
Just sanity check that the SQL under the hood is joining on the correct columns.